x-pack/plugins/observability/public/hooks/slo/mix_kql_with_tags.test.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { mixKqlWithTags } from './mix_kql_with_tags';
+
+describe('mixKQlQWithTags', () => {
+  it('mixes kql with selected tags', async () => {
+    const query = 'something';
+    const tags = {
+      included: ['production'],
+      excluded: [],
+    };
+
+    const kqlQueryMixedValue = mixKqlWithTags(query, tags);
+    expect(kqlQueryMixedValue).toBe('something and slo.tags: (production)');
+  });
+
+  it('mixes kql with cleared out tags', async () => {
+    const query = 'something';
+    const tags = {
+      included: [],
+      excluded: [],
+    };
+
+    const kqlQueryMixedValue = mixKqlWithTags(query, tags);
+    expect(kqlQueryMixedValue).toBe('something');
+  });
+
+  it('mixes kql with included and excluded tags', async () => {
+    const query = 'something';
+    const tags = {
+      included: ['production'],
+      excluded: ['dev'],
+    };
+
+    const kqlQueryMixedValue = mixKqlWithTags(query, tags);
+    expect(kqlQueryMixedValue).toBe('something and slo.tags: (production) and not slo.tags: (dev)');
```

Comment:
This fails locally it gives me the opposite thing `something and slo.tags: (dev) and not slo.tags: (production)`. I need to check if there is an actual problem in the implementation or in my test

Reply:
Fixed it!